src/voice-pe/using-the-device/wake-sound-toggle.md

```diff
@@ -7,7 +7,7 @@ zendesk:
 ---
 
 {% steps %}
-{% step "Toggling the wake sound switch" %}
+{% step "Toggling the wake sound switch" "Step" %}
```

Comment:
This does not match the PR title. Did you miss a commit?

Reply:
no. the PR title was inaccurate. Better?